packages/passport-auth/lib/Passport.js

```diff
@@ -76,7 +81,7 @@ class PassportAuthStrategy {
         }
         resolve({
           id: profile.id,
-          username: profile.username || null,
+          username: profile.username || profile.displayName || `${profile.name.givenName} ${profile.name.familyName}`,
```

Comment:
assuming `profile.name` is never null!

Reply:
Every service returns a different set of data for the name, username and displayName. So we decide to return the whole profile object and use just the service ID in this case.